src/averageAges.js

```diff
@@ -15,11 +15,27 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men = people.filter(person => person.sex === 'm');
+
+  if (century) {
+    men = men.filter(person => person.sex === 'm'
+      && Math.ceil(person.died / 100) === century);
```

Comment:
```suggestion
    men = men.filter(person => Math.ceil(person.died / 100) === century);
```
you already filter it by this condition